src/utils.ts

```diff
@@ -0,0 +1,50 @@
+import { Readable } from 'stream';
+import { Block } from './types';
+
+export function prependZeroes(number: number) {
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart

Reply:
@khorolets where is this function used? Do we need to have a dedicated function for this or is it fine to just inline it? Currently this function is really useless as "prependZeros" does not answer the question "why do I need to do it? and how much will it prepend? why should it prepend to make it exactly 12 characters?"

Reply:
@khorolets I suggest to rename this function to something that reflects the actual meaning, e.g. `normalizeBlockHeight` as currently this function is really confusing as "prependZeros" does not answer the question "why do I need to do it? and how many zeros will it prepend? why should it prepend to make it exactly 12 characters?"